app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_comments_counter_for_post
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- **[OPTIONAL]** Just like we have in the user modal, `after_save :update_comments_counter`